### PR TITLE
During upgrade stop services started by swift-init 

### PR DESCRIPTION
**Pull request by ryshah**
Remove service file not being used

### CODE REVIEW

**Comment by ryshah** [comment]
retest

**Comment by j2sol** [comment]
Looks good to me

File: upgrade.yml [hunk 1]
```diff
@@ -418,6 +418,14 @@
   any_errors_fatal: true
   tags: swift
 
+  pre_tasks:
+    # FIXME Added since we are changing swift init scripts in 3.0.x
+    # This will stop services started by "old" scripts. This can be removed
+    # after all systems have migrated to 3.0.x
+    - name: stop swift services started by swift-init
+      command: swift-init stop all
+      failed_when: false
```
Comment (craigtracey) [inline upgrade.yml]: why are we using failed_when vs. ignore_errors?

Comment (craigtracey) [inline upgrade.yml]: I am not confident that this is robust without ignore_errors

Comment (ryshah) [inline upgrade.yml]: In PR that I had related to swift upstart changes, @j2sol had recommended using failed_when: false instead of ignore_errors: true

Comment (j2sol) [inline upgrade.yml]: ignore_errors: true results in red colored output if there is a conditional that would normally error the task. This makes it hard to debug runs that have real fatal errors.

failed_when: false  does not color the output, but still ignores every error. Using `failed_when` is a style choice for Ursula.

**Pull request merged**